crates/isograph_schema/src/magic_mutation_fields.rs

```diff
@@ -27,13 +33,23 @@ lazy_static! {
     static ref FROM_VALUE_KEY_NAME: ValueKeyName = "from".intern().into();
     static ref TO_VALUE_KEY_NAME: ValueKeyName = "to".intern().into();
 }
-
+#[derive(Deserialize, Eq, PartialEq, Debug)]
 pub struct MagicMutationFieldInfo {
+    #[serde(deserialize_with = "deserialize_stringliteral")]
     path: StringLiteralValue,
     field_map: Vec<FieldMapItem>,
+    #[serde(deserialize_with = "deserialize_stringliteral")]
     field: StringLiteralValue,
 }
 
+fn deserialize_stringliteral<'de, D>(deserializer: D) -> Result<StringLiteralValue, D::Error>
```

Comment:
I added a macro rather than implementing it manually, what do you think? I originally implemented manually because haven't realized StringLiteralValue is a trait that belongs to a crate within isograph